react-frontend/src/components/SelectInput/SelectInput.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './SelectInput.css'
+
+const SelectInput = ({ label, value, onChange }) => {
+  return (
+    <div className="select-input">
+      <label>{label}</label>
```

Comment:
Please, use `for` attribute or wrap the entire control

Reply:
Thanks, refactored in [0c2832f](https://github.com/YaroslavKSE/AuctionWeb/pull/2/commits/0c2832f95bc0ca4eb2cb355a368e619b1378eb2b)